src/main.cpp

```diff
@@ -635,8 +635,6 @@ void setup()
 #if !MESHTASTIC_EXCLUDE_I2C
 #if !defined(ARCH_PORTDUINO) && !defined(ARCH_STM32WL) && !MESHTASTIC_EXCLUDE_ENVIRONMENTAL_SENSOR
     if (acc_info.type != ScanI2C::DeviceType::NONE) {
-        config.display.wake_on_tap_or_motion = true;
```

Comment:
We may consider adding this to the RAK4631 default config, since the WisMesh pocket comes with it. It's kind of a neat feature to enable OOTB. 😄

Reply:
i actually moved these to the default config for t-watch, feel free to add there.